package.json

```diff
@@ -18,16 +18,15 @@
     "@babel/core": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
-    "@mate-academy/scripts": "^0.9.2",
     "@mate-academy/students-ts-config": "*",
     "@types/jest": "^26.0.7",
     "@types/node": "^12.20.18",
     "babel-jest": "^26.1.0",
     "typescript": "^4.3.5"
   },
   "devDependencies": {
-    "@mate-academy/scripts": "^0.9.2",
     "@mate-academy/eslint-config": "*",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.